package com.kodilla.exception.test;

public class SecondChallengeMain {
    public static void main(String args[]) {
        ExceptionHandling exceptionHandling = new ExceptionHandling();
        exceptionHandling.testFunction();
    }
}
